### PR TITLE
My Code Review Suggestions.

### PR DESCRIPTION
**Pull request by sapravl**

### CODE REVIEW

**Comment by sapravl** [comment]
Q1.Summarize the three technical changes you made, and why.

1. On index.cshtml I removed the warning '100% is not a valid value of attribute width' by replacing 100% with equivalent
page size 1150. 

2. I updated the HTML format by adding end tags for the image element. As per best practices, empty elements must always be closed.
https://www.w3schools.com/html/html_xhtml.asp

3. I updated a couple of Variable names for better description and understanding.

4. Placeholder attribute added in search input element for better usage of the search bar space when not populated. 


Q2.List three items you learned from the code review.

1.I learned and understood various best practices that must be followed while writing code.

2.I learned how to make appropriate use of branches in GitHub.

3.I learned the usage of various elements and attributes of code.

Q3.Was the project available on GitHub on time?
Yes. The project was available on time.

Q4.Does the project compile?
Yes. The project compiles effectively with no errors.

Q5.Does the project have adequate documentation?
Yes, the project has sufficient documentation for use.

File: TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml [hunk 1]
```diff
@@ -19,19 +19,19 @@
...
-            <img src="~/images/1.png" width=100% height="350">
+            <img src="~/images/1.png" width=1150 height="350" />
         </div>
         <div class="carousel-item active text-center" data-interval="2000">
-            <img src="~/images/4.png" width=100% height="350">
+            <img src="~/images/4.png" width=1150 height="350" />
         </div>
         <div class="carousel-item" data-interval="2000">
-            <img src="~/images/6.jpg" width=100% height="350">
+            <img src="~/images/6.jpg" width=1150 height="350" />
```
Comment (sapravl) [inline TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml:31]: On index.cshtml removed the warning '100% is not a valid value of attribute width' by replacing 100% with equivalent page width of 1150.

File: TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml [hunk 2]
```diff
@@ -19,19 +19,19 @@
...
-                                <img src="~/images/MicrosoftTeams-image (16).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (16).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (17).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (17).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/MicrosoftTeams-image (18).png" accesskeywidth="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (18).png" accesskeywidth="300" height="200" />
```
Comment (sapravl) [inline TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml:34]: Updated the HTML format by adding end tags for the image element. As per best practices, empty elements must always be closed.
https://www.w3schools.com/html/html_xhtml.asp

File: TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml [hunk 3]
```diff
@@ -55,19 +55,19 @@
...
-                                <img src="~/images/MicrosoftTeams-image (21).png" width="300" height="200">
+                                <img src="~/images/MicrosoftTeams-image (21).png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/a.png" width="300" height="200">
+                                <img src="~/images/a.png" width="300" height="200" />
                             </div>
                             <div class="carousel-item" data-interval="2000">
-                                <img src="~/images/b.png" accesskeywidth="300" height="200">
+                                <img src="~/images/b.png" accesskeywidth="300" height="200" />
```
Comment (sapravl) [inline TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml:70]: I have updated this for all image elements in the Project.

File: TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs [hunk 4]
```diff
@@ -39,8 +39,8 @@ public void OnPost()
                     }
 
                 IDictionary<long, TheModernIlluminati.Models.Nobel> allNobels = new Dictionary<long, TheModernIlluminati.Models.Nobel>();
-                string nobelJSON = webClient.DownloadString("http://api.nobelprize.org/v1/laureate.json");
-                TheModernIlluminati.Models.Nobel nobel = TheModernIlluminati.Models.Nobel.FromJson(nobelJSON);
+                string nobelLaureateDetails = webClient.DownloadString("http://api.nobelprize.org/v1/laureate.json");
+                TheModernIlluminati.Models.Nobel nobel = TheModernIlluminati.Models.Nobel.FromJson(nobelLaureateDetails);
                 List<TheModernIlluminati.Models.Laureate> laureate1 = nobel.Laureates;
```
Comment (sapravl) [inline TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs:44]: Updated a couple of Variable names for better description and understanding.

File: TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs [hunk 5]
```diff
@@ -25,8 +25,8 @@ public void OnPost()
            
             using (var webClient = new WebClient())
             {
-                String countryJSON = webClient.DownloadString("http://api.nobelprize.org/v1/country.json");
-                    Country country = Country.FromJson(countryJSON);
+                String countryList = webClient.DownloadString("http://api.nobelprize.org/v1/country.json");
+                    Country country = Country.FromJson(countryList);
                     List<Count> random1 = country.Countries;
```
Comment (sapravl) [inline TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs:30]: Updated a couple of Variable names for better description and understanding.

File: TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml [hunk 6]
```diff
@@ -13,7 +13,7 @@
     <form action="" method="POST" style="margin-top: 50px;">
         <div class="form-group">
             <label style="font-size:20px;" class="control-label" for="CountrySearch"><strong>Enter Country</strong></label>
-            <input type="text" class="form-control" name="CountrySearch" id="CountrySearch" style="width:50%" />
+            <input type="text" class="form-control" placeholder="Please Enter Country to start search..." name="CountrySearch" id="CountrySearch" style="width:50%" />
         </div>
```
Comment (sapravl) [inline TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml:17]: Placeholder attribute added in search input element for better usage of the search bar space when not populated.

File: TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml [hunk 7]
```diff
@@ -19,19 +19,19 @@
     </ol>
     <div class="carousel-inner">
         <div class="carousel-item" data-interval="2000">
-            <img src="~/images/7.gif" width=100% height="350">
+            <img src="~/images/7.gif" width=1150 height="350" />
```
Comment (discospiff) [inline TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml:22]: Several notes:
1) relative positioning is preferred to absolute.  1150 is very device dependent.
2) Attribute values are not quoted, but should be.
3) Styling should be in linked CSS.
4) Image name is not descriptive, and there is no alt text.

File: TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml (hunk 1 above)
Comment (discospiff) [reply TheModernIlluminati/TheModernIlluminati/Pages/Index.cshtml:31]: See notes above.

File: TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml (hunk 2 above)
Comment (discospiff) [reply TheModernIlluminati/TheModernIlluminati/Pages/Laureate.cshtml:34]: This is a good find.  Also see notes above... there's other technical debt that needs to be addressed here.

File: TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml (hunk 6 above)
Comment (discospiff) [reply TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml:17]: Placeholder is a nice trick.  With that, you can remove label.

File: TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs (hunk 5 above)
Comment (discospiff) [reply TheModernIlluminati/TheModernIlluminati/Pages/SearchCountry.cshtml.cs:30]: Actually, the variable country makes more sense to be countryList, as country indicates one country, but there are actually several.

**Comment by discospiff** [comment]
From this review...

I wouldn't change relative positioning to absolute, but it does draw attention to some items in the image tags that should be remedied.  See my notes in Files Changed for more details.

**Pull request closed without merging**